src/stmt.h

```diff
@@ -66,7 +66,10 @@ class Stmt : public ASTNode {
     // also provide a default implementation of Optimize(), since most
     // Stmts don't have anything to do here.
     virtual Stmt *Optimize();
+
     virtual Stmt *TypeCheck() = 0;
+
+    virtual void SetLoopAttribute(std::pair<Globals::pragmaUnrollType, int>) = 0;
```

Comment:
Why not defining a default behavior (not accepting the pragma) and redefining it only for loop statments?

Reply:
Done